ansible/group_vars/all.yml

```diff
@@ -0,0 +1,124 @@
+---
+domain_name: ofa.iol.unh.edu
+domain_name_servers:
+  - 10.12.2.254
+network_prefix: 172.31
+lab_networks: 10.12.0.0/16
+ib0_2k_nets:
+  - ib0
+  - ib0.2
+ib0_4k_nets:
+  - ib0.4
+  - ib0.6
+ib0_nets: "{{ ib0_2k_nets + ib0_4k_nets }}"
+
+# InfiniBand Fabric 2
```

Comment:
Should these commented out configs stay?

Reply:
I'm not sure. Looking at the git history of rdma-functions.sh, it seems like these commented out configs have never been used but maybe they could be useful in the future.